File: src/com/mapdigit/game/microedition/drop/DropGamePlay.scala
package com.mapdigit.game.microedition.drop

import com.guidebee.game.graphics.{SVGImage, Texture}
import com.guidebee.game.{GameEngine, GamePlay}
import GameEngine._
import com.guidebee.game.audio.{Music, Sound}

class DropGamePlay extends GamePlay {

  override def create() {
    // load the assets

    assetManager.load("droplet.png", classOf[Texture])
    assetManager.load("bucket.png", classOf[Texture])

    assetManager.load("drop.wav", classOf[Sound])
    assetManager.load("fly.wav", classOf[Sound])
    assetManager.load("rain.mp3", classOf[Music])
    assetManager.load("lion.svg",classOf[SVGImage])

    assetManager.finishLoading()

    val scene = new MainScene()
    setScreen(scene)
  }

  override def dispose(): Unit ={
    assetManager.dispose()

  }

}

File: src/com/mapdigit/game/tutorial/drop/DropGamePlay.java
package com.mapdigit.game.tutorial.drop;

import com.guidebee.game.GamePlay;
import com.guidebee.game.audio.Music;
import com.guidebee.game.audio.Sound;
import com.guidebee.game.graphics.Texture;

import static com.guidebee.game.GameEngine.assetManager;

public class DropGamePlay extends GamePlay{
    @Override
    public void create() {
        loadAssets();
        DropScene screen=new DropScene();
        setScreen(screen);
    }

    @Override
    public void dispose(){
        assetManager.dispose();
    }

    private void loadAssets(){
        assetManager.load("droplet.png",Texture.class);
        assetManager.load("mario2.png",Texture.class);
        assetManager.load("bucket.png",Texture.class);
        assetManager.load("drop.wav",Sound.class);
        assetManager.load("rain.mp3",Music.class);
        assetManager.finishLoading();
    }
}
